### PR TITLE
docs: update wording for clone.md

### PR DESCRIPTION
**Pull request by CaitinChen**

### CODE REVIEW

File: docs/en/guide/clone.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 
-Please note that system tools like disk-free or disk-usage (`df`, `du`) will report the space used by the cloned data, but the underlying object storage space will not grow as blocks remains the same. On the same way, as metadata is actually replicated, the clone will take the same metadata engine storage space as the original.
+Note that system tools like disk-free or disk-usage (`df`, `du`) show the space used by the cloned data, but the underlying object storage space remains unchanged since the blocks are not duplicated. Cloning replicates metadata, so it will occupy the same metadata engine storage space as the original.
 
-**Clones takes up both file system storage space, inodes and metadata engine storage space**. Pay special attention when making clones on large size directories.
+**Cloning impacts both file system storage space, inodes and metadata engine storage space.** Be cautious when cloning large directories.
```
Comment (CaitinChen) [inline docs/en/guide/clone.md:15]: Why use "both"?
Does it mean "Cloning impacts file system storage space (including inodes) and metadata engine storage space."?

Comment (CaitinChen) [reply docs/en/guide/clone.md:15]: @xiaogaozi PTAL

Comment (yuhr123) [reply docs/en/guide/clone.md:15]: I guess it want to express the two parts of "storage" and "metadata".
The text can be rearranged to improve the expression, and whether to use "both" does not seem to be crucial.

Comment (CaitinChen) [inline docs/en/guide/clone.md:15]:
```suggestion
**Cloning impacts file system storage space, inodes, and metadata engine storage space.** Be cautious when cloning large directories.
```

**Pull request merged**